kokoro/gcp_ubuntu/continuous.cfg

```diff
@@ -0,0 +1,17 @@
+# -*- protobuffer -*-
```

Comment:
Please add license headers to all source code files.

Reply:
Added the license headers